cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -25,13 +27,100 @@
 ], scope="module")
 
 
-@pytest.mark.uncollectif(lambda: current_version() < '5.8')
 @pytest.mark.parametrize('context', [ViaSSUI])
-def test_ssui_dashboard(setup_provider, context, order_catalog_item_in_ops_ui):
-    """Tests various Primary and aggregate card values displayed on dashboard."""
+def test_total_services(setup_provider, context, order_catalog_item_in_ops_ui):
+    """Tests total services displayed on dashboard."""
+
+    appliance = get_or_create_current_appliance()
```

Comment:
**REQUIRED:** This is not needed, ```appliance``` is a global fixture now, so

```python
def test_total_services(appliance, setup_provider, context, order_catalog_item_in_ops_ui):
```

is good enough